python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -277,7 +277,10 @@ def _getCollectionArguments(workspace, reuse):
     # skip-existing-in would work around that, but would lead to a worse bug in
     # the case that the user is alternating runs with and without --clean-run.
     # registry.refresh()
-    oldRuns = list(registry.queryCollections(re.compile(workspace.outputName + r"/\d+T\d+Z")))
+    collectionPattern = re.compile(workspace.outputName + r"/\d+T\d+Z")
+    oldRuns = list(registry.queryCollections(workspace.outputName + "/*"))
```

Comment:
Minor comment but does it make things faster or slower if we use a more constrained glob such as `/????????T*Z` or `/*T*Z`?

Reply:
This is all happening in a small, temporary repo, so I don't think performance is an issue.